### DISCUSSION TITLE
Renaming Projects?

### DISCUSSION THREAD

**Discussion by skostojohn** [ransome1/sleek#722] [Q&A]
Is this possible in Sleek 2.x?

In searching the Issues, it seems like this was a feature in Sleek 1.x - can we do it in 2.x?  

Apologies if this is staring me in the face but I can't figure out how to do it.

Thanks!
Scott

**Comment by ransome1** [comment]
Hi Scott. Thanks for reaching out. Unfortunately this feature did not yet make it to version 2. Unfortunately development is very slow for some time now and I cannot really say when it will pick up again unfortunately.

As a workaround I can only recommend you to use a text editor and search and replace the projects, or simply use sleek 1.3.z for that tsk. I know that's not nice.

**Comment by ransome1** [comment]
> Is this possible in Sleek 2.x?
> 
> In searching the Issues, it seems like this was a feature in Sleek 1.x - can we do it in 2.x?
> 
> Apologies if this is staring me in the face but I can't figure out how to do it.
> 
> Thanks! Scott

@skostojohn sorry for the super late reply. This function has been added again in the latest pre-release and I hope you can find some time to test it: https://github.com/ransome1/sleek/releases/tag/v2.0.27-rc.1
